jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java

```diff
@@ -1193,6 +1199,7 @@ public String toString() {
     try {
       return getMetadataInJsonFunc();
     } catch (IoTDBSQLException e) {
+      LOGGER.info("Failed to fetch metadata in json because: ", e);
```

Comment:
LOGGER.error is better for Line 1202 1211 and 1220

Reply:
Thanks. I'll fix it.